src/rust/zh.manhuagui/res/source.json

```
@@ -0,0 +1,10 @@
+{
+	"info": {
+		"id": "zh.manhuagui",
+		"lang": "zh",
+		"name": "manhuagui",
+		"version": 1,
+		"url": "https://www.manhuagui.com/list/",
```

this should be the base url
```suggestion
		"url": "https://www.manhuagui.com/",
```